mathml/relations/css-styling/first-line-first-letter-pseudo-elements-004.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>MathML and ::first-line/::first-letter pseudo-elements</title>
+<link rel="help" href="https://w3c.github.io/mathml-core/#legacy-mathml-style-attributes">
+<link rel="match" href="first-line-first-letter-pseudo-elements-004-ref.html"/>
+<meta name="assert" content="::first-letter/::first-line works for HTML elements inside MathML.">
+<style>
+  .greenfirstline::first-line { background: lightgreen; }
```

Comment:
nit: they are now some inconsistencies for the class name "greenfirstline" VS "firstline" and color "lightgreen" VS "lime" with the other tests, it would be nice to make them consistent.